code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -169,17 +152,9 @@
 	icon = 'icons/obj/storage/closet.dmi'
 
 /obj/structure/closet/secure_closet/security/science/PopulateContents()
-	new /obj/item/radio/headset/headset_sci(src)
-	new /obj/item/clothing/shoes/sneakers/black(src)
-	new /obj/item/clothing/under/rank/security/officer/blueshirt/skyrat(src)
-	new /obj/item/clothing/head/helmet/blueshirt/skyrat(src)
-	new /obj/item/clothing/head/beret/sec/science(src)
-	new /obj/item/clothing/suit/armor/vest/blueshirt/skyrat(src)
 	new /obj/item/restraints/handcuffs/cable/pink(src)
```

Comment:
oh my god the guards get color-coded zipties since when

Reply:
It's really cute yeah.